### DISCUSSION TITLE
I can't submit my application get GitHub profile name error again and again

### DISCUSSION THREAD

**Discussion by Tillman245** [community/community#45518] [GitHub Education]
![Screenshot_20230127-160230](https://user-images.githubusercontent.com/123722326/215066107-ae426247-6144-4bfa-8566-6282f765fd85.png)

**Comment by jge162** [comment]
>I can't submit my application get GitHub profile name error again and again #45518

what profile are you referring to?

**Reply by jge162** [reply]
Education?

**Reply by jge162** [reply]
Ok I see the issue now. you are trying to get approved for education pack. Make sure you follow the instructions. Complete your profile with your name, last name etc. and upload a new picture as you have already used the one you are trying to submit now. @Tillman245
